Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -146,12 +158,13 @@ public void UpdateHit(RaycastHit hit)
                 };
             }
 
-            public void UpdateHit(RaycastResult result, RaycastHit hit)
+            public void UpdateHit(RaycastResult result, RaycastHit hit, RayStep sourceRay, int rayStepIndex)
```

Comment:
same here with the overload and marking obsolete.

Reply:
That could be dangerous since the old UpdateHit function would result in incomplete information and could cause silent errors / misbehavior. Should we throw a NotImplementedException when it's called?

Reply:
Eg
```
[Obsolete]
public void UpdateHit(RaycastHit hit)
{
    throw new NotImplementedException();
}
```

Reply:
You won't have incomplete info if you provide the defaults.  but yeah throwing not implemented it fine.

Reply:
Can we give directions on the `Obsolete` attribute on how to fix it? The obsolete message will just show up in the unity logs without any comment on how to fix it otherwise. People have to go dig into the code to find out how (which is easy in this case since it's just the method below but still). 

If you'll throw an exception this should probably be a breaking change then?

Reply:
Agreed, the Obsolete should also tell us what to replace it's usage with.

Reply:
Added info to all Obsolete tags - let me know if these need to be more detailed.